src/components/sbb-icon/sbb-icon.scss

```diff
@@ -0,0 +1,32 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-icon-margin-inline: 0;
+  --sbb-icon-margin-block: 0;
+
+  display: inline-block;
+  line-height: 0;
+  margin-inline: var(--sbb-icon-margin-inline-override, var(--sbb-icon-margin-inline));
+  margin-block: var(--sbb-icon-margin-block-override, var(--sbb-icon-margin-block));
+}
+
+svg:not(.color-immutable) {
+  [fill]:not([fill='none']) {
+    fill: currentcolor;
+  }
+
+  [stroke]:not([stroke='none']) {
+    stroke: currentcolor;
+  }
+
+  /* stylelint-disable plugin/stylelint-bem-namics */
+  // TODO: Maybe improve this solution in CDN Image
+  & >,
+  :where(:not([stroke]):not([fill])) {
+    :where(path, polygon, polyline):where(:not([stroke]):not([fill])) {
+      fill: currentcolor;
+    }
+  }
```

Comment:
Enable again the rule at the end of the block with 
`/* stylelint-enable plugin/stylelint-bem-namics */`